freqtrade/strategy/interface.py

```diff
@@ -861,23 +862,22 @@ def min_roi_reached(self, trade: Trade, current_profit: float, current_time: dat
         else:
             return current_profit > roi
 
-    def ft_check_timed_out(self, side: str, trade: Trade, order: Dict,
+    def ft_check_timed_out(self, side: str, trade: LocalTrade, order: Order,
```

Comment:
Is `LocalTrade` typehint correct? This function is called in live too

Reply:
LocalTrade is the parent class of Trade (Trade derives from `LocalTrade`).